dist/header.hbs

```diff
@@ -1,8 +1,4 @@
-{{#if isPatch~}}
-  ###
-{{~else~}}
-  ##
-{{~/if}} {{#if @root.linkCompare~}}
+## {{#if @root.linkCompare~}}
```

Comment:
🤷

Reply:
I have yet to pay attention to why this repo outputs handlebars files, no worries 😉